Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -21,7 +26,9 @@ def gistify_logs(f)
       files["tap.out"] = { :content => tap }
     end
 
-    url = create_gist(files)
+    # Description formatted to work well as page title when viewing gist
+    descr = "#{f.name} on #{MACOS_FULL_VERSION} - Homebrew build logs"
```

Comment:
I suggest:
* `f.full_name` instead of `f.name`.
* `OS_VERSION` instead of `MACOS_FULL_VERSION`.